client/src/assets/scss/base.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * Base Styles
+ */
+
+@import "theme.scss";
+
+/* @font-face should go here */
+
+/* HTML element styles should go here */
+
+/* global styles should go here */
```

Comment:
Groundwork for the eventual migration to scss?

Reply:
Yes.

Reply:
What else can I say here? This project has some components using SASS and others using SCSS. It would be nice to be consistent between the Entities UIs.

Reply:
Absolutely!